### ISSUE TITLE
fetch_harvard_oxford quirks

### ISSUE THREAD

**Issue by AlexandreAbraham** [nilearn/nilearn#486]
Harvard Oxford fetcher:
- [x] does not return a Bunch in contrary to all the other fetch_ functions but a tuple
- [x] does not use the standard nilearn_data location to fetch the data but the current working directory instead (not reproducible on my side)
- [x] browse FSLDIR instead of browsing FSLDIR/data/atlases/
- [x] should look up in default system directories for each system
- [x] should mimic FSL data directory tree in nilearn_data.

**Comment by banilo** [comment]
Should I take care of this?

**Comment by lesteve** [comment]
> does not use the standard nilearn_data location to fetch the data but the current working directory instead (not reproducible on my side)

I can reproduce the problem even when NILEARN_DATA is set. Here is the output:

* NILEARN_DATA no set:
```python

In [1]: from nilearn import datasets

In [2]: datasets.fetch_harvard_oxford('cort-maxprob-thr25-2mm', verbose=10)
Dataset search paths: ['', '', '/home/le243287/nilearn_data']

Dataset created in harvard_oxford

Downloading data from https://www.nitrc.org/frs/download.php/7363/HarvardOxford.tgz ...
Downloaded 7879863 of 7879863 bytes (100.00%,    0.0s remaining)  
...done. (13 seconds, 0 min)
Extracting data from harvard_oxford/dc9b313e598e25514deb51b257475841/HarvardOxford.tgz...
   ...done.

```

* NILEARN_DATA set to /tmp:

```python
In [1]: from nilearn import datasets

In [2]: datasets.fetch_harvard_oxford('cort-maxprob-thr25-2mm', verbose=10)
Dataset search paths: ['', '', '/tmp', '/home/le243287/nilearn_data']

Dataset created in harvard_oxford

Downloading data from https://www.nitrc.org/frs/download.php/7363/HarvardOxford.tgz ...
Downloaded 7879863 of 7879863 bytes (100.00%,    0.0s remaining)  
...done. (9 seconds, 0 min)
Extracting data from harvard_oxford/dc9b313e598e25514deb51b257475841/HarvardOxford.tgz...
   ...done.
```

In both cases for some reason `''` is inserted in the dataset search paths twice before the usual paths.

**Comment by AlexandreAbraham** [comment]
OK, this is related to FSLDIR resolution. This is an easyfix

**Comment by lesteve** [comment]
> Should I take care of this?

@banilo would make sense since you have worked on this fetcher in the past IIRC.

**Comment by AlexandreAbraham** [comment]
If somebody could take care of that for 0.1.3 release, that would be great!

**Comment by salma1601** [comment]
fetch_harvard_oxford reloads or not my existant atlas, depending on what my current working directory is!

**Comment by AlexandreAbraham** [comment]
Yes, this is a side effect of the different quirks. I'll try to set up a quickfix today.

**Comment by salma1601** [comment]
Ok thanks !

**Comment by AlexandreAbraham** [comment]
Fixed by #597.
